### ISSUE TITLE
 Can't find methods such as fit and transform in scikit-cuda PCA 

### ISSUE THREAD

**Issue by woodenwatcher** [lebedov/scikit-cuda#272]
If haven't those method, the code as follow will not working

  pca.fit(X)
  pca.transform(Y)

**Comment by nmerrill67** [comment]
Yes, we only have a fit_transform method currently. Although fit and transform separately would be a nice addition.

**Comment by woodenwatcher** [comment]
will you add these method in the near future?

**Comment by nmerrill67** [comment]
@woodenwatcher I have been trying on and off since I first wrote the code to figure out how to do this. It is not immediately obvious from the original paper linked in the skcuda PCA docs. So long story short, yes -- but no success so far. I will update if any success.

**Comment by fjgmoya** [comment]
@nmerrill67, once fit_transform generates P_gpu array with principal components, my question is if a **transform** method could be ("easily") created multiplying (Dgemm) R_gpu matrix (n*p) and P_gpu one (p*k) in order to generate a "S_gpu" matrix (n*k) afected by Lambda, and get the desired result. Thank you in advance.

**Comment by ashok-arjun** [comment]
Any update on implementing these methods?

**Comment by MrRace** [comment]
Any plan to add transform function?  @nmerrill67  As we know the PCA in `from sklearn.decomposition import PCA`  has the `transform` function  indeed.  Looking forward to this function~
